src/resolvers/get-make.ts

```diff
@@ -311,7 +311,39 @@ function getMakeFromVin(vin: string): Make | null {
     case '9BS':
       return Make.SCANIA
     case 'VSS':
-      return Make.SEAT
+      switch (modelCode) {
```

Comment:
If the VIN starts with `VSSZZZKM` or `VSSZZZK1`, they should also always be mapped to CUPRA (even without model code present): https://app.shortcut.com/connectedcars/story/95805/resolve-cupra-brand-for-relevant-cars#activity-95812